### PR TITLE
feat: add forgot password modal

### PR DESCRIPTION
**Pull request by bangunbagustapa**
### Changes

- Change error response that was thrown
- Add forgot password modal
- Add action to reset password link

### Preview

https://user-images.githubusercontent.com/55199533/148016395-4766a5ae-05f3-4cff-836c-3e26990e7c07.mov

### Evidence

Title: add forgot password modal
Project: Portal Jabar
Participants: @bangunbagustapa @adzharamrullah @maulanayuseph @Ibwedagama

### CODE REVIEW

File: src/components/Login/ForgotPassword.vue [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<template>
+  <BaseModal :open="open">
+    <div class="w-full h-full px-2 pb-4">
+      <h1 class="font-roboto font-medium text-green-700 text-[21px] leading-[34px] mb-6">
+        Lupa Kata Sandi
+      </h1>
+      <div class="flex gap-8">
+        <div class="w-[68px] h-[55px]">
+          <img src="@/assets/icons/mailbox.svg">
```
Comment (Ibwedagama) [inline src/components/Login/ForgotPassword.vue:9]: hi @bangunbagustapa I think it's better to add `alt, width and height attribute to the image

**Review by adzharamrullah — APPROVED** [review]

File: src/components/Login/ForgotPassword.vue [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    return {
+      email: '',
+      error: null,
+      success: false,
+    };
+  },
+  computed: {
+    isValidInput() {
+      return this.email !== '' && this.isValidEmail(this.email) && this.password !== '';
+    },
```
Comment (Ibwedagama) [inline src/components/Login/ForgotPassword.vue:122]: hi @bangunbagustapa , is this line ``` && this.password !== ' '``` necessary?

**Review by maulanayuseph — APPROVED** [review]

File: src/components/Login/ForgotPassword.vue (hunk 1 above)
Comment (bangunbagustapa) [reply src/components/Login/ForgotPassword.vue]: I forgot to add those, changes here https://github.com/jabardigitalservice/portal-jabar-cms/pull/15/commits/25bfc92c3abee4be4512b4df45d728afebeaec42

File: src/components/Login/ForgotPassword.vue (hunk 2 above)
Comment (bangunbagustapa) [reply src/components/Login/ForgotPassword.vue]: My bad, iuse this code from previous component and didn't check it. Changes here https://github.com/jabardigitalservice/portal-jabar-cms/pull/15/commits/842e2c8b17ed23e7fa7d93d166ffbef304032ebf. Thank you @Ibwedagama

**Review by Ibwedagama — APPROVED** [review]

**Pull request merged**